src/tests/ForecastSummary.test.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastSummary from "../components/ForecastSummary";
+
+describe("ForecastSummary", () => {
+  const validProps = {
+    date: 1111111,
+    description: "Stub description",
+    icon: "800",
+    temperature: {
+      min: 12,
+      max: 22,
+    },
+    onSelect: () => {},
+  };
+
+  it("renders correctly", () => {
+    const { asFragment } = render(
+      <ForecastSummary
+        date={validProps.date}
+        description={validProps.description}
+        icon={validProps.icon}
+        temperature={validProps.temperature}
+        onSelect={validProps.onSelect}
+      />
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("render correct values for props", () => {
+    const { getByText, getByTestId } = render(
+      <ForecastSummary
+        date={validProps.date}
+        description={validProps.description}
+        icon={validProps.icon}
+        temperature={validProps.temperature}
+        onSelect={validProps.onSelect}
+      />
+    );
+
+    expect(getByText("January 1st")).toHaveClass("forecast-summary__date");
```

Comment:
There is a bug in the test here, "January 1st" is not what is rendered on screen